package.json

```diff
@@ -1,13 +1,13 @@
 {
-    "name": "brite-rest",
+    "name": "eventbrite",
```

Comment:
why not something like eventbrite-sdk-javascript

Reply:
Barrett wanted `eventbrite`. Having `javascript` in the name is pretty redundant considering it's a node module. And `sdk` is pretty much implied?

Reply:
I thought brite-rest was pretty funny. that being said my first attempt to import it I tried it as `eventbrite`